src/main/webapp/scripts/place-suggestions.js

```diff
@@ -1,5 +1,7 @@
 let city;
 let filter;
+const placeCardsContainer = document.getElementById('place-cards-container');
+const defaultPlaceImage = '../assets/img/jason-dent-blue.jpg'
```

Comment:
Since this is a shared constant, maybe use the capitals + snake_case formatting instead of camelCase

Reply:
Good style suggestion, will fix.